lib/aws/awsconfig.js

```diff
@@ -20,6 +20,73 @@ function AWSConfig() {
     return configService;
   }
 
+  me.enableRule = function(input) {
+    var self = arguments.callee;
+    var configService = me.findService(input);
+    if(input.owner == "CUSTOM_LAMBDA"){
+    var params = {
```

Comment:
How about this?

`var params = {`

    ConfigRule:{
        ConfigRuleName:input.ruleName,
        Description:input.descript,
        Source: {
            Owner: input.owner,
            SourceIdentifier: input.sourceID,
        },
        Scope: {ComplianceResourceTypes: input.resourceType},
        InputParameters: input.params
    }    
`};`

`if(input.owner == "CUSTOM_LAMBDA"){`

  `params.ConfigRule.Source.SourceDetails = [`

      {
          "EventSource": "aws.config",
          "MessageType": "ConfigurationItemChangeNotification"
      }
  `]`

 `}`